Project.toml

```diff
@@ -17,13 +17,13 @@ Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 
 [compat]
 FEMQuad = "0.3"
-FastGaussQuadrature = "0.4, 0.5"
+FastGaussQuadrature = "0.4, 0.5, 1.0"
 ForwardDiff = "0.10"
 LinearSolve = "1, 2"
 NearestNeighbors = "0.4"
-StaticArrays = "1.2"
-Symbolics = "5"
-julia = "1.6"
+StaticArrays = "1.9"
```

Comment:
I think `StaticArrays = "1.2"` allows for "1.9" doesn't it? Do you want to force the minimum version of SA to be 1.9 because of some feature?

Reply:
right!